### ISSUE TITLE
[io] Explore the access to remote HDF5 files

### ISSUE THREAD

**Issue by vasole** [silx-kit/silx#1120]
Try to use the h5serv: https://github.com/HDFGroup/h5serv

**Comment by vallsv** [comment]
The project is well done. Very easy to compile and execute.

Here is the instructions
```
virtualenv h5serv.env
source h5serv.env/bin/activate
pip install pip --upgrade
pip install wheel setuptools --upgrade

git clone https://github.com/HDFGroup/h5serv.git h5serv.git
cd h5serv.git
pip install .

cd server
python app.py

xfd-open http://127.0.0.1:5000/
```

**Comment by vallsv** [comment]
I am not able to have a working server. For information i follow there Travis commands https://github.com/HDFGroup/h5serv/blob/develop/.travis.yml

**Comment by vallsv** [comment]
Works much better without the proxy...
```
unset https_proxy
unset http_proxy
```

**Comment by vasole** [comment]
closed by #1140
